master/upgrade-to-1.0.md

```diff
@@ -0,0 +1,85 @@
+# Upgrading to Velero 1.0
+
+## Prerequisites
+- Velero v0.11 installed. If you're not already on v0.11, see the [instructions for upgrading to v0.11][0]. **Upgrading directly from v0.10.x or earlier to v1.0 is not supported!**
```

Comment:
Should we be even more explicit and required `v0.11.1`, so the `migrate-backups` command is present?

Reply:
The nuance there is that they don't need to be running v0.11.1 on the server, they just need the v0.11.1 client.  That's why i left this as just v0.11, and included the v0.11.1 CLI download instructions below. WDYT?

Reply:
That makes sense. In that case it can be left.